### DISCUSSION TITLE
Slots

### DISCUSSION THREAD

**Discussion by CavalcanteLeo** [formkit/formkit#1222] [General]
Is it possible to use scoped slots defined by components? Because our input is a vue component, so we would like to use its slots like this

```vue
<FormKit type="myInput">
  <template #customSlot="slotProps">
     ...
  </template>
</FormKit>
```

```ts
createSection('input', () => ({
$cmp: 'MyInput',
}))

```

**Comment by justin-schroeder** [comment]
You can, all the slots are available inside the context object: https://formkit.link/8fc5988df21b8f28909b5df44b238e43
